### DISCUSSION TITLE
`lineHeight` of Label with FormattedString

### DISCUSSION THREAD

**Discussion by mr-luke** [NativeScript/NativeScript#10018] [Q&A]
Hello everyone 👋🏼 

I develop a news app that displays paragraphs full of bolded, linked and italic words. Here is my problem. When I use a `Label` with `FormattedString`, I cannot set the `lineHeight` which is crucial for a readability of text. Do you have any idea how to deal with it?

I'm using Nativescript `8.3.x` with Vue.

Thank U in advance 🙏🏼

**Comment by farfromrefug** [comment]
@mr-luke are you saying setting `lineHeight` on the `Label` wont work while using `FormattedString` or are you saying you want to use `lineHeight ` on a per `Span` basis?

**Reply by mr-luke** [reply]
I'm saying that property `lineHeight` set on the `Label` with `FormattedString` inside doesn't work.

**Comment by vallemar** [comment]
Hi! can you pass for here your template with label, formattedString... or link in preview 2 with the problem

**Comment by mr-luke** [comment]
The top level of Page looks like this:
```
<ScrollView orientation="vertical">
  <StackLayout class="article-view">
    // Here goes an iteration that construct an article from modules
  </StackLayout>
</ScrollView>
```

A Paragraph module looks like this:
```
<Label class="module module__paragraph" :textWrap="true" :lineHeight="8">
  <FormattedString>
    <Span v-for="(p, i) in textParts" :key="i" v-bind="p.decoration" :text="p.content" />
  </FormattedString>
</Label>
```

This is a part of more complex structure but more or less when I use this structure all decorations work fine but `lineHeight` is gone.
When I use unparsed content (I mean with `text` on `Label` without `FormattedString`), `lineHeight` works perfectly 🤔

**Reply by farfromrefug** [reply]
@mr-luke could you try to replace `Label` with `HTMLLabel` (drop-in replacement) from `@nativescript-community/ui-label` ? If it works i can simply create a PR to fix N `Label` component.
I think i already see the issue
